corehq/apps/accounting/async_handlers.py

```diff
@@ -250,6 +250,9 @@ class Select2CustomerInvoiceTriggerHandler(BaseSelect2AsyncHandler):
     @property
     def customer_account_response(self):
         accounts = BillingAccount.objects.filter(is_customer_billing_account=True).values_list('name', flat=True)
+        if self. search_string:
```

Comment:
Nitpick: remove the space. (I assumed this was a syntax error and was surprised to learn it works.)